kork/kork-web/src/main/java/com/netflix/spinnaker/kork/client/ServiceClientProvider.java

```diff
@@ -25,7 +25,11 @@
 
 @NonnullByDefault
 public interface ServiceClientProvider {
-
+  /** Which version of retrofit to use. Retrofit 1.x and 2.x are supported. */
```

Comment:
Could you add some detail about how to decide which version to use?  It needs to match where the annotations come from in the interface class, right?

Reply:
Added